tokio/src/task/local.rs

```diff
@@ -242,6 +242,8 @@ impl LocalSet {
     ///
     /// This task is guaranteed to be run on the current thread.
     ///
+    /// Calls to this function will result in waking the current localset.
+    ///
```

Comment:
I'm not sure this is necessary to have in the doc?

Reply:
Happy to remove it if you don't think it is necessary. I added it because a side effect that changes program behavior is worth noting imo.

Reply:
I don't think it is necessary. The "wake" is an implementation detail.

Reply:
Besides, notes about changes in behavior go in the changelog instead. (You don't have to put it in this PR, we do them together when making the release.)

Reply:
Makes sense - Thanks for the review! Pushed the suggested changes.